### PR TITLE
UniParc light and cross reference

### PR DESCRIPTION
**Pull request by ahmadshadab**

### CODE REVIEW

**Review by LeonardoGonzales — COMMENTED** [review]
Can you please make sure that we clean UniParc voldemort classes?

File: integration-test/src/main/java/org/uniprot/store/indexer/uniparc/mockers/UniParcEntryMocker.java [hunk 1]
```diff
@@ -25,46 +27,104 @@ public class UniParcEntryMocker {
...
+            String id, int sequenceLength, List<UniParcCrossReference> crossReferences) {
+        UniParcEntryBuilder builder = new UniParcEntryBuilder();
+        StringBuilder sequence = new StringBuilder();
+        IntStream.range(0, sequenceLength).forEach(i -> sequence.append("A"));
+        Sequence uniSeq = new SequenceBuilder(sequence.toString()).build();
+        builder.uniParcId(id).sequence(uniSeq).uniParcCrossReferencesSet(crossReferences);
+        return builder.build();
+    }
+
+    public static UniParcEntryLight createUniParcEntryLight(
```
[the remark this answers is not in the document]
Comment (LeonardoGonzales) [reply integration-test/src/main/java/org/uniprot/store/indexer/uniparc/mockers/UniParcEntryMocker.java:61]: This is missing. If you do not think it is necessary it is okay.

File: uniprot-config/src/test/java/org/uniprot/store/config/returnfield/config/impl/UniParcReturnFieldConfigImplIT.java [hunk 2]
```diff
@@ -117,7 +109,9 @@ private static Stream<Arguments> provideReturnSortFields() {
 
     private static Stream<Arguments> provideReturnFieldNames() {
         return returnFieldConfig.getReturnFields().stream()
+                .filter(rf -> Objects.nonNull(rf.getChildNumber()))
                 .map(ReturnField::getName)
+                .filter(name-> !name.equals("common_taxons"))
```
Comment (LeonardoGonzales) [inline uniprot-config/src/test/java/org/uniprot/store/config/returnfield/config/impl/UniParcReturnFieldConfigImplIT.java:114]: We should not filter common_taxons here. If it is failing with it, it means there is an issue. We also have to implement in for TSV, maybe this is the issue.

Comment (ahmadshadab) [reply uniprot-config/src/test/java/org/uniprot/store/config/returnfield/config/impl/UniParcReturnFieldConfigImplIT.java]: Supun has added config test based on UniParcLightEntry

**Review by LeonardoGonzales — APPROVED** [review]

**Pull request merged**